src/sections/SMP/index.js

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+
+import Layout from "../../components/layout"; 
+
+import SMPWrapper from "./smp.style";
+import { Row, Col, Container } from "../../reusecore/Layout";
+import Button from "../../reusecore/Button";
+
+import example from "./service-mesh-performance-specification-example.gif";
+import c_icon from "./c-icon.svg";
+import wasm_capacity from "./native-and-wasm-at-capacity-100rps.png";
+import latency from "./latency-at-scale.png";
+import client_capacity from "./client-capacity.png";
+
+const SMPPage = () => {
+    return (
+        <Layout>
+            <SMPWrapper>
+                <div className="smp-hero">
+                    <Container>
+                        <div className="hero-text">
+                            <Row>
+                                <Col lg={7}>
+                                    <h1> Service Mesh Performance </h1>
+                                    <p> The Service Mesh Performance Working Group is hosted within CNCF SIG Network. All are welcome to participate. This group is defining the Service Mesh Performance(SMP). Using SMP, MeshMark provides a universal performance index to gauge your mesh's efficiency against deployments in other organizations' environments.</p>
+                                </Col>
```

Comment:
What would we add in the right part of hero section. Also we haven't used SMP logo on the entire page yet.

Reply:
I think the SMP logo can go there 🤔

Reply:
Use of the logo there makes much sense to me as well.

Reply:
The SMP logo does not suit with the background here.